streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -24,7 +24,13 @@
 import org.apache.kafka.streams.processor.StreamPartitioner;
 
 /**
- * KStream is an abstraction of a <i>record stream</i> of key-value pairs.
+ * {@link KStream} is an abstraction of a <i>record stream</i> of key-value pairs.
+ * <p>
+ * A {@link KStream} is either defined from one or multiple Kafka topics that are consumed message by message or
+ * the result of a {@link KStream} transformation. A {@link KTable} can also be converted into a {@link KStream}.
```

Comment:
How about add {@see KTable} since we mentioned it in the Javadoc? Vice versa for KTable.